docs/site-example/i18n/de.po

```diff
@@ -28,35 +28,63 @@ msgstr ""
 msgid "gluon-config-mode:pubkey"
 msgstr ""
 "<p>Dies ist der öffentliche Schlüssel deines Freifunk-Knotens. Erst nachdem "
-"er auf den Servern des Freifunk-Projektes auf Alpha Centauri eingetragen wurde, "
-"kann sich dein Knoten mit dem Mesh-VPN dort verbinden. Bitte "
-"schicke dazu diesen Schlüssel und den Namen deines Knotens "
+"er auf den Servern des Freifunk-Projektes auf Alpha Centauri eingetragen "
+"wurde, kann sich dein Knoten mit dem Mesh-VPN dort verbinden. Bitte schicke "
+"dazu diesen Schlüssel und den Namen deines Knotens "
 "(<em><%=pcdata(hostname)%></em>) an "
-"<a href=\"mailto:keys@alpha-centauri.freifunk.net?subject=<%= urlencode('Anmeldung: ' .. hostname) %>&amp;"
-"body=<%= urlencode('# ' .. hostname .. '\n' .. pubkey) %>\">keys@alpha-centauri.freifunk.net</a>."
+"<a href=\"mailto:keys@alpha-centauri.freifunk.net?subject="
+"<%= urlencode('Anmeldung: ' .. hostname) %>&amp;body="
+"<%= urlencode('# ' .. hostname .. '\n# ' .. sysconfig.primary_mac .. '\nkey ') %>"
+"%22<%= pubkey %>%22;"
+"<%= urlencode('\n\nIch habe zur Kenntnis genommen, dass der im ') %>"
```

Comment:
not every community publishes the contact. the only thing which applies to every community i think would be, that the contact can be seen by anyone using the specific Freifunk network (via respondd)

Reply:
I thought, that everyone can ask respondd for the contact in batman anyway, also if the contact is not shown on the map.

Reply:
sure, but respondd doesn't answer to public ipv6 requests - and even if it does, not every community has public ipv6

Reply:
Ich denke eher, dass der text so gemeint war, dass die contact adresse dort halt von dritten abgefragt werden kann, die auch im Freifunk Netz sind

Reply:
ok, i changed it.